hw/ip/adc_ctrl/dv/README.md

```diff
@@ -53,7 +53,7 @@ The ADC data interface is emulated using push_pull agents. ([`push_pull_agent`](
 ### UVM RAL Model
 The ADC_CTRL RAL model is created with the [`ralgen`](../../../dv/tools/ralgen/README.md) FuseSoC generator script automatically when the simulation is at the build stage.
 
-It can be created manually by invoking [`regtool`](../../../../util/reggen/doc/setup_and_use.md):
+It can be created manually by invoking [`regtool`](../../../../util/reggen/doc/setup_and_use.md).
```

Comment:
Suggest to separate this fix for all modules other than I2C into a separate PR and also change the template at `util/uvmdvgen/index.md.tpl`. Other parts of that template may have to be fixed too, e.g., the `relrefs` I think no longer work with our new documentation renderer? This would benefit from separate reviewing & tracking

Reply:
Good idea. I'll just leave the I2C change in here and create a new PR for the `uvmdvgen` changes.

Reply:
I've created a draft PR to track this activity: https://github.com/lowRISC/opentitan/pull/18470